mlflow/protos/unity_catalog_oss_messages.proto

```diff
@@ -21,6 +21,73 @@ message CreateRegisteredModel {
   }
 }
 
+message DeleteRegisteredModel {
+  // Required. Full name of the model.
+  optional string full_name = 1 [(validate_required) = true];
+  // Optional. Force deletion even if the registered model is not empty. Default is false.
+  optional bool force = 2;
+
+  message Response {}
+}
+
+message GetRegisteredModel {
+
+  // Required. Full name of the Model.
+  optional string full_name = 1 [(validate_required) = true];
+
+  message Response {
+    optional RegisteredModelInfo registered_model_info = 1 [(json_inline) = true];
+  }
+}
+
+message CreateModelVersion {
+  optional ModelVersionInfo model_version_info = 1 [(json_inline) = true];
```

Comment:
The API for create model version looks like this: 
https://docs.databricks.com/api/workspace/registeredmodels/create

The UC OSS definition is this:
```
    CreateRegisteredModel:
      type: object
      properties:
        name:
          description: Name of model, relative to parent schema.
          type: string
        catalog_name:
          description: Name of parent catalog.
          type: string
        schema_name:
          description: Name of parent schema relative to its parent catalog.
          type: string
        comment:
          description: User-provided free-form text description.
          type: string
      required:
        - name
        - catalog_name
        - schema_name
        ```